deps/internal/util/common.go

```diff
@@ -0,0 +1,53 @@
+package util
+
+import (
+	"net/url"
+	"strings"
+
+	"github.com/depscloud/api/v1alpha/schema"
+)
+
+// SetModuleFields parses "Name" into Org and Module
+func SetModuleFields(module *schema.Module) *schema.Module {
```

Comment:
This is done server side now. Should be no need to do it on the client.

Reply:
The reason why I had to clone this logic on `deps` is, to translate `destinationModuleName` into a `schema.Module`. As I've written right now, we don't ever pass this on to server, instead pass only the start node.

And then parse the results of DFS on the start node to find paths that leads to `destinationModuleName`. For this matching to be done on the client side, I needed these util methods.